content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Control Topology Management Policies on a node
+reviewers:
+- lmdaly
+- nolancon
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+An increasing number of systems leverage a combination of CPUs and hardware accelerators to support latency-critical execution and high-throughput parallel computation. These include workloads in fields such as telecommunications, scientific computing, machine learning, financial services and data analytics. Such hybrid systems comprise a high performance environment.
+
+In order to extract the best performance, optimizations related to CPU isolation and memory and device locality are required. However, in Kubernetes, these optimizations are handled by a disjoint set of components.
+
+Topology Manager is a component in Kubelet that provides node level policies to enable these performance optimizations in an user abstract manner.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Topology Manager
+
+Currently, the CPU and Device Manager in Kubernetes make resource allocation decisions independently of each other.
+This can result in a sub-optimal allocations on multiple socketed systems, performance/latency sensitive applications
+will suffer due to these sub-optimal allocations.
+
+The Topology Manager introduces a algorithm to allow components that care about Topology Aware resource allocation to provide
+a list of their resources with the associated socket. This allows the Topology Manager to calculate the best possible socket for
+the incoming container and store this. The resource allocators can then consult this when making their resource allocation decisions.
+
+### Configuration
+
+The Topology Manager is an alpha feature in Kubernetes v1.15. 
+
+The Topology Manager currently works only on a node with the `static` CPU Manager
+Policy enabled and on Pods in the Guaranteed QOS. Topology Manager will align CPU and device requests.
```

Comment:
```suggestion
Policy enabled and on Pods in the `Guaranteed` {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}. Topology Manager will align CPU and device requests.
```
(maybe re-wrap lines if accepting?)

Reply:
@sftim thanks for the review. Could you show me how I could link the following page in this document as well using the tooltips?
https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/

Reply:
I don't think there's a glossary entry for that concept (which is fine; it's a glossary, not an index).
For that, you can use a Markdown hyperlink, eg
```markdown
something something [control CPU management policies](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/) on the node something something
```

Reply:
There's a section called `whatsnext` in several pages on this website, you could add a What's Next section to this page and link there.

See https://github.com/kubernetes/website/blame/8bf13937df0bb36de602f7dd6ac06b7f8867f433/content/en/docs/concepts/overview/what-is-kubernetes.md#L206 for an example of this block.